envoy/config/filter/http/jwt_authn/v2/config.proto

```diff
@@ -0,0 +1,260 @@
+
+syntax = "proto3";
+
+package envoy.config.filter.http.jwtauthn.v2;
```

Comment:
Is missing underscore intentional here: `jwt_authn`?

Reply:
I also wonder this, this seems to violate our wish to have package name and path match. @rodaine has pointed out in the past that we can run into Go issues at Lyft if we violate this.